### DISCUSSION TITLE
Will automerge only merge one PR at a time?

### DISCUSSION THREAD

**Discussion by caphrim007** [renovatebot/renovate#8221] [Request Help] [answered]
I have a renovate bot running on a cron schedule that appears to only automerge one PR at a time regardless of how many are ready to be merged. I'm wondering if this is expected behavior or not.

When it runs, it outputs work done on one PR, then it mentions (for example) the remaining 2, but then proceeds to exit as if saying it's completed the work it was expected to do. Is it because of the depName being the same in the package rules?

**Comment by viceice** [comment, marked as answer]
Renovate will ever only automerge one pr, because it will only automerge when the branch is up-to-date with target branch. So after automerge all other branches need to be updated.

For your second question we need more information. Eg your specific config.

**Reply by caphrim007** [reply]
here's a (sanitized) link to it https://gist.github.com/caphrim007/8c6d26a327340e8bfb9db45484e15194

however I suspect what you're describing is what is happening. maybe the config would confirm that??

**Reply by viceice** [reply]
yes, it breaks futher update processing after automerge as expected.
